config/accounts.go

```diff
@@ -131,6 +131,18 @@ func (a *AccountGDPR) PurposeEnforced(purpose consentconstants.Purpose) (value,
 	return *a.PurposeConfigs[purpose].EnforcePurpose, true
 }
 
+// PurposeEnforcementAlgo checks the purpose enforcement algo for a given purpose by first
+// looking at the account settings, and if not set there, defaulting to the host configuration.
+func (a *AccountGDPR) PurposeEnforcementAlgo(purpose consentconstants.Purpose) (value string, exists bool) {
```

Comment:
I am a bit confused as to what an algp is in this context and why the value is always returned as TCF2FullEnforcement.

Reply:
This is a mistake. The function now returns whichever algorithm (full or basic) is set on the account, and if it isn't set, it returns the default full.